packages/env/lib/build-docker-compose-config.js

```diff
@@ -227,11 +246,20 @@ module.exports = function buildDockerComposeConfig( config ) {
 				volumes: [ 'mysql-test:/var/lib/mysql' ],
 			},
 			wordpress: {
-				build: '.',
+				build: {
+					context: '.',
+					args: {
+						HOST_USERNAME: hostUser.name,
+						HOST_UID: hostUser.uid,
+						HOST_GID: hostUser.gid,
+					},
+				},
 				depends_on: [ 'mysql' ],
 				image: developmentWpImage,
 				ports: [ developmentPorts ],
 				environment: {
+					APACHE_RUN_USER: '#' + hostUser.uid,
+					APACHE_RUN_GROUP: '#' + hostUser.gid,
```

Comment:
The `'#'` is necessary to get Apache to recognize this as an ID instead of a name.